.circleci/config.yml

```diff
@@ -18,15 +18,24 @@ references:
         - checkout-content
         - prepare-tox3-7
 
+    create_requirement_files: &create_requirement_files
+      run:
+        name: Create requirement files
+        command: |
+          pip install pipfile-requirements
+          pipfile2req > requirements.txt
+          pipfile2req -d > requirements-dev.txt
+
 jobs:
   tox3-7-unit-tests:
       # docker image circleci/python:3.8.3-buster-node also contains python 3.7
       docker:
         - image: circleci/python:3.8.3-buster-node
       steps:
         - checkout
+        - *create_requirement_files
         - restore_cache:
-            key: tox-37-cache-{{ checksum "requirements.txt" }}-{{ checksum "requirements-dev.txt" }}
+            key: tox-37-cache-{{ checksum "Pipfile" }}-{{ checksum "Pipfile.lock" }}
```

Comment:
create a reference to that key, exactly like you did with `create_requirements_files`. tox-37 is widely used. you can keep the tox-38 cache as is.